File: loto.cs
using pt;

	public class textoss{
		public static void Main(string[] args){
			int n;
			pt.pts.apagar();
			for(n=0;n<16;n++){
				pt.pts.localiza(0,n);
				pt.pts.escrever(pt.pts.texton((int) pt.pts.loto(1,50)));
			}
		}
	}

File: loto.vb
imports pt
module modelo1
	sub main()
			dim n as integer
			pt.pts.apagar()
			for n=0 to 16 
				pt.pts.localiza(0,n)
				pt.pts.escrever(pt.pts.texton(pt.pts.loto(1,50)))
			next

	end sub
end module
